frontend/src/index.scss

```diff
@@ -424,9 +787,9 @@ header .navbar.navbar-static-top {
 
               }
 
-              // programs
-              & > :nth-child(3) {
-              }
+              // // programs
+              // & > :nth-child(3) {
+              // }
```

Comment:
That could have been deleted instead of being commented out

Reply:
I did not want to delete that comment because it contains the info that the 3rd child are the programs. It is probably a bit useless, as nobody would not trust a commented out comment to be still correct. I will remove the comment in a second PR that includes other minor fixes for the dark theme.